pkg/excmd/executor.go

```diff
@@ -102,16 +102,16 @@ func (e *executor) exec(c *Command, cmd *exec.Cmd) (out []byte, err error) {
 		wg.Add(2)
 		go func() {
 			defer wg.Done()
-			io.Copy(e.io.Out(), io.TeeReader(outReader, &buf))
+			io.Copy(e.io.Out, io.TeeReader(outReader, &buf))
 		}()
 		closers = append(closers, outReader.Close)
 		go func() {
 			defer wg.Done()
-			io.Copy(e.io.Err(), io.TeeReader(errReader, &buf))
+			io.Copy(e.io.Err, io.TeeReader(errReader, &buf))
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Error return value of `io.Copy` is not checked (errcheck)